_includes/checklist.md

```diff
@@ -1,41 +1,91 @@
-* **Main repository:** <url>
-* **Running libraries:**
-    * <url>
-    * ...
-* **Site:** <url>
-* **Product manager:** @<username>
-* **Technical point of contact:** @<username>
+* **Main repository:** [url]
+* **Site:** [url]
+* **Product manager:** @[username]
+* **System Owner:** @[username of technical point of contact who will be the project representative in the Sprint]
+* **Infrastructure Lead:** @[username]
+* **ATO folder:** [url]
+* **Sprint notes:** [url]
 
 ## TODOs
 
-### Project team
+### ATO Sprint prerequisites
 
-Many of these tasks can be done in parallel. This is just a suggestion of priority.
+#### Infrastructure Lead
 
+- [ ] Set up an ATO intro meeting with the team.
+- [ ] Determine the [impact level](https://pages.18f.gov/before-you-ship/ato/levels/).
+- [ ] Add this issue to the `Backlog` of [the ATO Kanban board](https://github.com/18F/Infrastructure/projects/1).
+- [ ] Assign the appropriate labels to this issue.
+- [ ] Set up the project ATO folder.
+    - [ ] Create as a sub-folder of [the `ATOs` folder in Google Drive](https://drive.google.com/a/gsa.gov/folderview?id=0BynIxtx-CfkdckljM3BPSkdQT1U&usp=sharing) under 18F/OPP/PIF then "Work in progress". Link to it above.
```

Comment:
Good idea. Personally I find "<project name> ATO - <duration> <level>" to be easy to read.